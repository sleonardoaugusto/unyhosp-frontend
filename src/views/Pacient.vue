<template>
  <div>Pacient Detail</div>
</template>

<script>
export default {
  name: 'Pacient'
}
</script>

<style scoped>

</style>