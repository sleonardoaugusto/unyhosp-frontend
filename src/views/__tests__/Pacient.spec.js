import { shallowMount, createLocalVue } from '@vue/test-utils'
import Pacient from '@/views/Pacient'
import VueRouter from 'vue-router'

const localVue = createLocalVue()
localVue.use(VueRouter)

describe('Pacient.vue', () => {
  let wrapper

  wrapper = shallowMount(Pacient, {
    localVue
  })

  it('renders component properly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})