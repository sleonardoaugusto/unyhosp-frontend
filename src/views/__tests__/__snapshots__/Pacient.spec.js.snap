// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pacient.vue renders component properly 1`] = `
<div class="col-md-12">
  <div class="row d-flex justify-content-center">
    <h1 class="type-h2 type-blue-smoke">Paciente</h1>
  </div>
  <router-view-stub name="default"></router-view-stub>
</div>
`;
